xbmc/interfaces/json-rpc/FileItemHandler.cpp

```diff
@@ -359,7 +359,7 @@ void CFileItemHandler::HandleFileItem(const char *ID, bool allowFile, const char
     if (item->HasEPGInfoTag())
       FillDetails(item->GetEPGInfoTag().get(), item, fields, object, thumbLoader);
     if (item->HasPVRRecordingInfoTag())
-      FillDetails(item->GetPVRRecordingInfoTag(), item, fields, object, thumbLoader);
+      FillDetails(item->GetPVRRecordingInfoTag().get(), item, fields, object, thumbLoader);
```

Comment:
yeah like I said, I was just asking why the .get() call was needed instead of passing the shared_ptr